style/main.scss

```diff
@@ -1,30 +1,13 @@
-@import "@primer/css/color-modes/themes/light.scss";
-@import "@primer/css/color-modes/themes/dark.scss";
+@import "./node_modules/github-markdown-css/github-markdown.css";
 
 .markdown-body {
-  @import "@primer/css/support/index.scss";
-  @import "@primer/css/base/index.scss";
-
-  background-color: var(--color-canvas-default);
-  color: var(--color-fg-default);
-
-  ol {
-    list-style: decimal;
-  }
-  
-  ul {
-    list-style: disc;
-  }
```

Comment:
This is added in f4c447091d1fc527cea475c06b1ad6579c987049 and actually not necessary.

I assume you looked at a website with CSS reset (e.g. [Tailwind's Preflight](https://tailwindcss.com/docs/preflight)) and made the wrong assumption that they are needed.

Reply:
Why is this not needed? If you are using this on a site with Tailwind's preflight, won't the list style be unset?

Reply:
https://github.com/denoland/dotland/pull/2040#issuecomment-1057151477